src/sign.builds

```diff
@@ -10,8 +10,20 @@
 
   <UsingTask AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.dll" TaskName="ReadSigningRequired" />
 
+  <ItemGroup>
+    <WindowsNativeLocation Include="$(NativeBinDir)\clrcompression.dll" />
+    <WindowsNativeLocation Include="$(NativeBinDir)_aot\clrcompression.dll" />
+  </ItemGroup>
+  
+  <Target Name="GenerateSignForWindowsNative">
+    <WriteSigningRequired AuthenticodeSig="$(AuthenticodeSig)"
+                          StrongNameSig="$(null)"
```

Comment:
I don't think $(null) is doing what you think it is doing. I assume you can just remove this property completely as it is not a required property.